src/main/java/uk/gov/hmcts/divorce/bulkscan/transformation/ApplicationTransformer.java

```diff
@@ -81,9 +84,21 @@ private void setMarriageBrokenDetails(TransformationDetails transformationDetail
                 "Please review confirmation of breakdown for joint application in the scanned form"
             );
         }
-        caseData.getApplication().setApplicant1ScreenHasMarriageBroken(
-            from(toBoolean(ocrDataFields.getSoleOrApplicant1ConfirmationOfBreakdown()))
-        );
+
+        if (toBoolean(ocrDataFields.getSoleOrApplicant1ConfirmationOfBreakdown())) {
+            if (caseData.isDivorce()) {
+                caseData.getApplication().setApplicant1HasMarriageBroken(Set.of(MARRIAGE_BROKEN));
+            } else {
+                caseData.getApplication().setApplicant1HasCivilPartnershipBroken(Set.of(CIVIL_PARTNERSHIP_BROKEN));
+            }
+        } else {
+            if (caseData.isDivorce()) {
+                caseData.getApplication().setApplicant1HasMarriageBroken(emptySet());
+            } else {
+                caseData.getApplication().setApplicant1HasCivilPartnershipBroken(emptySet());
+            }
```

Comment:
do we need to set it to empty if the confirmation is not ticked on the form ?

Reply:
discussed and will leave it set to emptySet incase it is called upon elsewhere in the application to avoid any chance of a NPE.